perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/PerunBlImpl.java

```diff
@@ -118,6 +118,7 @@
 
 	private final static Set<String> dontLookupUsersForLogins = BeansUtils.getCoreConfig().getDontLookupUsers();
 	private final static Set<String> extSourcesWithMultipleIdentifiers = BeansUtils.getCoreConfig().getExtSourcesMultipleIdentifiers();
+	private final static String shibIdentityProvider_key = "originIdentityProvider";
```

Comment:
You can use one public static String identifier, instead create a duplicate of it.

Reply:
Fixed